### ISSUE TITLE
Sinatra::Capture possible encoding issues

### ISSUE THREAD

**Issue by zzak** [sinatra/sinatra#1144]
This ticket is copied from sinatra/sinatra-contrib#75

**Comment by jkowens** [comment]
@namusyaka I don't think this is an issue any longer since we're not using `render` for capture. I believe it is safe to close.
